nanogpt/basic_sampler.h

```diff
@@ -0,0 +1,25 @@
+#include <vector>
```

Comment:
This file is no longer needed since we put the greedy sampling inside the forward function.